src/main/resources/zeebe-payments.json

```diff
@@ -27,6 +27,16 @@
               "type": "text"
             }
           }
+        },
+        {
```

Comment:
This was added for flush warning for some records which was passing a object rather than a string/text.